Cargo.toml

```diff
@@ -1,16 +1,40 @@
 [package]
 name = "hades252"
-version = "0.1.0"
-authors = ["kev <kevtheappdev@gmail.com>"]
+version = "0.1.1"
+authors = [
+  "kev <kevtheappdev@gmail.com>", "zer0 <matteo@dusk.network>",
+  "Victor Lopes <victor@dusk.network>", "kr0 <c.perezbaro@gmail.com>"
+]
 edition = "2018"
 
 [dependencies]
 hex = "0.3.1"
 sha2 = "0.8"
-curve25519-dalek = { version = "1.0.1", features = ["serde"] }
-merlin = "1.1"
+curve25519-dalek = { version = "1.2.3", features = ["serde", "avx2_backend"] }
+merlin = "1.2.0"
 rand = "0.6"
+lazy_static = "1.3.0"
 
 [dependencies.bulletproofs]
-path = "./bulletproofs"
+git = "https://github.com/dusk-network/bulletproofs"
```

Comment:
This can be compressed to: `bulletproofs = { git = "https://github.com/dusk-network/bulletproofs", branch = "develop" }`

Maybe could also be included as a normal dependency (idk)

Reply:
I prefer to keep it separate, especially because it's our fork: it's more readable and it's easier to spot immediately – compare to the other deps.

Reply:
Perfect. I also agree now that you mention this.